### ISSUE TITLE
failed to install firefox-debug-adapter

### ISSUE THREAD

**Issue by JSteitz** [williamboman/mason.nvim#134]
### Problem description

It's the same issue as #41 

### Neovim version (>= 0.7)

NVIM v0.7.2
Build type: Release
LuaJIT 2.1.0-beta3

### Operating system/version

Linux soda 5.15.55-1-lts #1 SMP Fri, 15 Jul 2022 11:28:54 +0000 x86_64 GNU/Linux

### I've recently downloaded the latest plugin version of mason.nvim

- [x] Yes

### Affected packages

firefox-debug-adapter

### Actual behavior

Failing

### Expected behavior

Installing

### Mason output

```Text
Failed
    ◍ firefox-debug-adapter
      Cloning into '.'...
      From https://github.com/firefox-devtools/vscode-firefox-debug
       * tag               2.9.8      -> FETCH_HEAD
      Note: switching to 'FETCH_HEAD'.
      
      You are in 'detached HEAD' state. You can look around, make experimental
      changes and commit them, and you can discard any commits you make in this
      state without impacting any branches by switching back to a branch.
      
      If you want to create a new branch to retain commits you create, you may
      do so (now or later) by using -c with the switch command. Example:
      
        git switch -c <new-branch-name>
      
      Or undo this operation with:
      
        git switch -
      
      Turn off this advice by setting config variable advice.detachedHead to false
      
      HEAD is now at 6ad01e3 Merge pull request #294 from firefox-devtools/2.9.8
      patching file webpack.config.js
      Hunk #2 FAILED at 27.
      1 out of 2 hunks FAILED -- saving rejects to file webpack.config.js.rej
      spawn: patch failed with exit code 1 and signal 0.
```

### Installation log

```Text
[DEBUG Mon 25 Jul 2022 11:05:33 AM CEST] .../site/pack/packer/start/mason.nvim/lua/mason-core/fs.lua:70: fs: mkdirp /home/jura/.local/share/nvim/mason/.packages/firefox-debug-adapter
[DEBUG Mon 25 Jul 2022 11:05:33 AM CEST] ...te/pack/packer/start/mason.nvim/lua/mason-core/fetch.lua:32: Fetching URL "https://latest-github-tag.redwill.se/api/repo/firefox-devtools/vscode-firefox-debug/latest-tag"
[DEBUG Mon 25 Jul 2022 11:05:33 AM CEST] .../pack/packer/start/mason.nvim/lua/mason-core/process.lua:115: Spawning cmd="wget", spawn_opts={
  args = { "--header='User-Agent: mason.nvim (+https://github.com/williamboman/mason.nvim)'", "-nv", "-O", "-", "--method=GET", "https://latest-github-tag.redwill.se/api/repo/firefox-devtools/vscode-firefox-debug/latest-tag" }
}
[DEBUG Mon 25 Jul 2022 11:05:33 AM CEST] .../pack/packer/start/mason.nvim/lua/mason-core/process.lua:161: Spawned with pid 16164
[DEBUG Mon 25 Jul 2022 11:05:33 AM CEST] .../pack/packer/start/mason.nvim/lua/mason-core/process.lua:147: Job pid=16164 exited with exit_code=0, signal=0
[DEBUG Mon 25 Jul 2022 11:05:33 AM CEST] .../pack/packer/start/mason.nvim/lua/mason-core/process.lua:115: Spawning cmd="git", spawn_opts={
  args = { "clone", "--depth", "1", "https://github.com/firefox-devtools/vscode-firefox-debug", "." },
  cwd = "/home/jura/.local/share/nvim/mason/.packages/firefox-debug-adapter"
}
[DEBUG Mon 25 Jul 2022 11:05:33 AM CEST] .../pack/packer/start/mason.nvim/lua/mason-core/process.lua:161: Spawned with pid 16165
[DEBUG Mon 25 Jul 2022 11:05:33 AM CEST] .../pack/packer/start/mason.nvim/lua/mason-core/process.lua:147: Job pid=16165 exited with exit_code=0, signal=0
[DEBUG Mon 25 Jul 2022 11:05:33 AM CEST] .../pack/packer/start/mason.nvim/lua/mason-core/process.lua:115: Spawning cmd="git", spawn_opts={
  args = { "fetch", "--depth", "1", "origin", "2.9.8" },
  cwd = "/home/jura/.local/share/nvim/mason/.packages/firefox-debug-adapter"
}
[DEBUG Mon 25 Jul 2022 11:05:33 AM CEST] .../pack/packer/start/mason.nvim/lua/mason-core/process.lua:161: Spawned with pid 16186
[DEBUG Mon 25 Jul 2022 11:05:34 AM CEST] .../pack/packer/start/mason.nvim/lua/mason-core/process.lua:147: Job pid=16186 exited with exit_code=0, signal=0
[DEBUG Mon 25 Jul 2022 11:05:34 AM CEST] .../pack/packer/start/mason.nvim/lua/mason-core/process.lua:115: Spawning cmd="git", spawn_opts={
  args = { "checkout", "FETCH_HEAD" },
  cwd = "/home/jura/.local/share/nvim/mason/.packages/firefox-debug-adapter"
}
[DEBUG Mon 25 Jul 2022 11:05:34 AM CEST] .../pack/packer/start/mason.nvim/lua/mason-core/process.lua:161: Spawned with pid 16199
[DEBUG Mon 25 Jul 2022 11:05:34 AM CEST] .../pack/packer/start/mason.nvim/lua/mason-core/process.lua:147: Job pid=16199 exited with exit_code=0, signal=0
[DEBUG Mon 25 Jul 2022 11:05:34 AM CEST] .../pack/packer/start/mason.nvim/lua/mason-core/process.lua:115: Spawning cmd="patch", spawn_opts={
  args = { "-g", "0", "-f" },
  cwd = "/home/jura/.local/share/nvim/mason/.packages/firefox-debug-adapter"
}
[DEBUG Mon 25 Jul 2022 11:05:34 AM CEST] .../pack/packer/start/mason.nvim/lua/mason-core/process.lua:161: Spawned with pid 16210
[DEBUG Mon 25 Jul 2022 11:05:34 AM CEST] .../pack/packer/start/mason.nvim/lua/mason-core/process.lua:147: Job pid=16210 exited with exit_code=1, signal=0
[ERROR Mon 25 Jul 2022 11:05:34 AM CEST] ...acker/start/mason.nvim/lua/mason-core/installer/init.lua:146: Installation failed for Package(name=firefox-debug-adapter) error=spawn: patch failed with exit code 1 and signal 0. 
[DEBUG Mon 25 Jul 2022 11:05:34 AM CEST] .../site/pack/packer/start/mason.nvim/lua/mason-core/fs.lua:46: fs: rmrf /home/jura/.local/share/nvim/mason/.packages/firefox-debug-adapter
[DEBUG Mon 25 Jul 2022 11:05:34 AM CEST] ...ker/start/mason.nvim/lua/mason-core/installer/linker.lua:21: Unlinking Package(name=firefox-debug-adapter)
```

### Healthcheck

```Text
mason: require("mason.health").check()
========================================================================
## mason.nvim report
  - OK: neovim version >= 0.7.0
  - OK: **Go**: `go version go1.18.4 linux/amd64`
  - OK: **cargo**: `cargo 1.62.1`
  - WARNING: **luarocks**: not available
  - OK: **Ruby**: `ruby 3.0.4p208 (2022-04-12 revision 3fa771dded) [x86_64-linux]`
  - OK: **RubyGem**: `3.3.15`
  - OK: **Composer**: `Composer version 2.3.10 2022-07-13 15:48:23`
  - OK: **PHP**: `PHP 8.1.8 (cli) (built: Jul  9 2022 06:10:37) (NTS)`
  - OK: **npm**: `8.15.0`
  - OK: **node**: `v18.6.0`
  - OK: **python3**: `Python 3.10.5`
  - OK: **pip3**: `pip 22.1.2 from /usr/lib/python3.10/site-packages/pip (python 3.10)`
  - OK: **javac**: `Ok`
  - OK: **java**: `openjdk version "1.8.0_342"`
  - WARNING: **julia**: not available
  - OK: **wget**: `GNU Wget 1.21.3 built on linux-gnu.`
  - OK: **curl**: `curl 7.84.0 (x86_64-pc-linux-gnu) libcurl/7.84.0 OpenSSL/1.1.1q zlib/1.2.12 brotli/1.0.9 zstd/1.5.2 libidn2/2.3.3 libpsl/0.21.1 (+libidn2/2.3.0) libssh2/1.10.0 nghttp2/1.48.0`
  - OK: **gzip**: `gzip 1.12`
  - OK: **tar**: `tar (GNU tar) 1.34`
  - OK: **bash**: `GNU bash, version 5.1.16(1)-release (x86_64-pc-linux-gnu)`
  - OK: **sh**: `Ok`
  - OK: GitHub API rate limit. Used: 2. Remaining: 58. Limit: 60. Reset: Mon 25 Jul 2022 11:41:59 AM CEST.
```

**Comment by williamboman** [comment]
Ah, it seems like they fixed a macOS build issue upstream. I didn't get pinged so wasn't aware they'd done so, should be fixed in #135!

**Comment by JSteitz** [comment]
Really, that was way too fast :)
